### PR TITLE
14.0

### PR DESCRIPTION
**Pull request by pq**
@devoncarew @skybrian @mit-mit 

All: any changes we want to get in before I cut an RC for this release?

### CODE REVIEW

**Comment by skybrian** [comment]
Time flies. Nothing pending here. Looks like Devon has a pull request.

**Comment by pq** [comment]
Updated w/ a blip for #1044...

File: resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -22,6 +22,21 @@
...
 
+14.0:
+<ul>
+  <li>user toggleable option to enable more verbose debug logging of Flutter app runs</li>
+  <li>fixes to the new Flutter Module workflow</li>
+  <li>improved console logging on Flutter app termination</li>
+  <li>improved error reporting on Observatory connection and Flutter View open failures</li>
+  <li>removed (meaningless) Flutter SDK settings from default projects</li>
+  <li>improved project name validation (to align with checks in `flutter create`)</li>
+  <li>console hyperlinks for xcode resources</li>
```
Comment (devoncarew) [inline resources/META-INF/plugin.xml]: `XCode`

File: resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -22,6 +22,21 @@
   <change-notes>
     <![CDATA[
 
+14.0:
+<ul>
+  <li>user toggleable option to enable more verbose debug logging of Flutter app runs</li>
+  <li>fixes to the new Flutter Module workflow</li>
+  <li>improved console logging on Flutter app termination</li>
+  <li>improved error reporting on Observatory connection and Flutter View open failures</li>
+  <li>removed (meaningless) Flutter SDK settings from default projects</li>
```
Comment (devoncarew) [inline resources/META-INF/plugin.xml]: perhaps elide `(meaningless)`?

**Comment by devoncarew** [comment]
lgtm

**Comment by pq** [comment]
Comments addressed.  Thanks!

File: resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -28,9 +28,9 @@
   <li>fixes to the new Flutter Module workflow</li>
   <li>improved console logging on Flutter app termination</li>
   <li>improved error reporting on Observatory connection and Flutter View open failures</li>
-  <li>removed (meaningless) Flutter SDK settings from default projects</li>
+  <li>removed Flutter SDK settings from default projects</li>
   <li>improved project name validation (to align with checks in `flutter create`)</li>
-  <li>console hyperlinks for xcode resources</li>
+  <li>console hyperlinks for XCode resources</li>
```
Comment (mit-mit) [inline resources/META-INF/plugin.xml]: I believe the correct caps are just `Xcode`

Comment (pq) [inline resources/META-INF/plugin.xml]: Good catch.

**Pull request merged**